packages/components/tree/tree-tokens.scss

```
@@ -0,0 +1,93 @@
+.kbq-tree {
```

Такого селектора должно быть достаточно, чтобы не прокидывать токены в `tree-option.component.ts`
```suggestion
.kbq-tree, .kbq-tree-selection {
```